pkg/connect/client/clientset.go

```diff
@@ -8,6 +8,7 @@ import (
 	apbindingv1alpha1 "github.com/cofide/cofide-api-sdk/pkg/connect/client/apbinding/v1alpha1"
 	attestationpolicyv1alpha1 "github.com/cofide/cofide-api-sdk/pkg/connect/client/attestationpolicy/v1alpha1"
 	clusterv1alpha1 "github.com/cofide/cofide-api-sdk/pkg/connect/client/cluster/v1alpha1"
+	federationV1Alpha1 "github.com/cofide/cofide-api-sdk/pkg/connect/client/federation/v1alpha1"
```

Comment:
nit: import renames should be lower case: https://google.github.io/styleguide/go/decisions.html#package-names

Reply:
our linter should catch that then, no?

Reply:
Ideally, yes. I haven't found a plugin to enforce it yet though.